src/components/navigation/NavPills/components/OrientationWrapper.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Box from '@mui/material/Box'
+import { OrientationWrapperProps } from '../types'
+
+const OrientationWrapper: React.FC<OrientationWrapperProps> = ({ children, actions, orientation }): any => {
```

Comment:
same here as in NavPillsStyles.ts file, we need to specify the actual types of the properties  instead of  `any`